experimental/yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -133,6 +133,88 @@ class MapOperator : public FlowableOperator<U, D> {
   F function_;
 };
 
+template <
+    typename U,
+    typename D,
+    typename F,
+    typename = typename std::enable_if<std::is_trivially_assignable<D, U>::value>,
```

Comment:
I might add a "#if version" check in here so it can build for older gcc versions. Is it an appreciated way in the team for such features that are only available in the new compiler versions.